pages/apim/3.x/kubernetes/apim-kubernetes-operator-overview.adoc

```diff
@@ -0,0 +1,30 @@
+[[apim-kubernetes-operator-overview]]
+= Gravitee Kubernetes Operator (GKO)
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_kubernetes_operator_overview.html
+:page-folder: apim/kubernetes
+:page-layout: apim3x
+
+[label label-version]#New in version 3.19.0#
+
+== Overview
+
+APIM 3.19.0 has introduced the Gravitee Kubernetes Operator (GKO) - a new technical component designed to be deployed on a Kubernetes Cluster. It can also be deployed on a local cluster for testing purposes.
+
+You can use the GKO in the following ways:
+
+  * To define, deploy, and publish APIs to your API Portal and API Gateway and to manage Custom Resource Definitions (CRDs) as part of the process.
+  * To use the GKO as an Ingress Controller for deploying Ingresses to your API Gateway.
+  * To deploy Gravitee products such as API Management, Access Management, or Alert Engine.
+  * To improve automation processes through covering CICD aspects when using Kubernetes with APIM.
+  * To manage most API Management resources without directly relying on the Console or on the Management API.
```

Comment:
📝 
CRD for api definition allow to have infra as code for this part too and so to version it like any other K8S configuration.

Reply:
updated